scripts/update.js

```diff
@@ -7,16 +7,42 @@ const prettier = require("prettier");
 
 const PERMISSIONS_DOCUMENTATION_URL =
   "https://docs.github.com/en/free-pro-team@latest/rest/reference/permissions-required-for-github-apps/";
-const CACHE_FILE_PATH = "cache/api.github.com.html";
+const PERMISSIONS_DOCUMENTATION_CACHE_FILE_PATH = "cache/api.github.com.html";
+const APP_PERMISSIONS_SCHEMA_CACHE_FILE_PATH =
+  "cache/app-permissions-schema.json";
 const GENERATED_JSON_FILE_PATH = "generated/api.github.com.json";
 const DOCUMENTED_BASE_URL = "https://docs.github.com";
 
+// TODO: these should be added to GitHub's OpenAPI schema
+const MISSING_SCHEMA_PERMISSIONS = [
+  "codespaces",
+  "dependabot_secrets",
+  "email_addresses",
+  "followers",
+  "git_ssh_keys",
+  "gpg_keys",
+  "interaction_limits",
+  "organization_events",
+  "organization_webhooks",
+  "profile",
+  "repository_webhooks",
+  "self_hosted_runners",
+  "starring",
+];
```

Comment:
@kfcampbell @nickfloyd can you please look up the real names of these permissions and also reach out to the OpenAPI team and let them know that these permissions are missing in the schema at `components/schemas/app-permissions`

We can ship the PR as is and remove these later, or add our own mapping until the OpenAPI spec is updated.

Reply:
Hey @gr2m will do... thanks for the head's up!

Reply:
Updating here: I talked to the Apps team and they're receptive of a PR internally to add these permissions.